src/api/mod.rs

```diff
@@ -672,6 +692,18 @@ impl<T: Pixel> IntoFrame<T> for (Arc<Frame<T>>, Option<FrameParameters>) {
 }
 
 impl<T: Pixel> Context<T> {
+  /// Allocates and returns a new frame.
+  ///
+  /// # Examples
```

Comment:
`s/Examples/Example/`

Reply:
It is always `# Examples` and `# Panics`, even if one example. Check the standard library docs, e.g. https://doc.rust-lang.org/std/vec/struct.Vec.html#method.capacity

Reply:
Sounds pretty stupid to me - is this actully documted as "you need to do this for rustdocs" anywhere, or is it just "stdlib does this"?

Reply:
Not sure it's spelled out anywhere as a guideline, but as far as I know it's pretty standard. Besides, it means no changing the line if examples are added or removed, and (provided that you can link to subsections, never tried that actually) consistent linking if you want to link to an example.